Sources/DNA/Color/Color+Button.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+extension UIColor {
+    public class var callToActionButtonHighlightedBodyColor: UIColor {
```

Comment:
Not sure if we need a new class for this, would be nice to have all the colors in one single file

Reply:
I agree with @3lvis. Maybe keep it as a separate extension, but in the same file as the others. Also, I don't like the confusion in naming this file referencing `Color` which currently isn't a type. The existing definitions of colors is done in an extension on `UIColor`.